### PR TITLE
adds sauerkraut dish and recipe

### PR DESCRIPTION
**Pull request by RealestEstate**
2 units salt
15 units water
1 cabbage
:cl: 
* rscadd: adds sauerkraut dish

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5707,4 +5707,13 @@ obj/item/weapon/reagent_containers/food/snacks/butteredtoast
 	reagents.add_reagent(NUTRIMENT, 2)
 
 
+/obj/item/weapon/reagent_containers/food/snacks/sauerkraut
+	name = "sauerkraut"
+	desc = "Cabbage that has feremented in salty brine."
```
Comment (BarneyGumball) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: >feremented

**Comment by RealestEstate** [comment]
I'm ready to update the wiki once this is merged.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -5707,4 +5707,13 @@ obj/item/weapon/reagent_containers/food/snacks/butteredtoast
...
+/obj/item/weapon/reagent_containers/food/snacks/sauerkraut
+	name = "sauerkraut"
+	desc = "Cabbage that has fermented in salty brine."
+	icon_state = "sauerkraut"
+	trash = /obj/item/trash/snack_bowl
 
+/obj/item/weapon/reagent_containers/food/snacks/sauerkraut/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 1)
+	reagents.del_reagent(WATER)
```
Comment (DamianX) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Why is this here? Is WATER being added somewhere?
